astropy/coordinates/baseframe.py

```diff
@@ -1759,7 +1759,7 @@ def separation(
         Computes on-sky separation between this coordinate and another.
 
         For more on how to use this (and related) functionality, see the
-        examples in :doc:`astropy:/coordinates/matchsep`.
+        examples in :doc:`astropy:coordinates/matchsep`.
```

Comment:
Is this a viable alternative?
```suggestion
        examples in :ref:`astropy-coordinates-separations-matching`.
```

Reply:
Worth a try!

Reply:
👍🏻

Reply:
That seems to work here and with my affiliated package example.

Reply:
I think `:ref:` to explicit section like this would satisfied all the requirements mentioned in https://github.com/astropy/astropy/pull/16758#issuecomment-2248785741 .